source/includes/_wallet.md

```diff
@@ -772,7 +805,7 @@ curl $url/wallet/$id/shared-key \
 
 ```javascript
 const httpWallet = new bcoin.http.wallet({ id: id });
-const account = 'default';
+account = 'default';
```

Comment:
`const` could have been retained.

Reply:
It's declared further up now in the javascript declarations with `id` and `key` to be consistent, so it shouldn't need the const anymore.

Reply:
Ah, missed that.